.github/workflows/ci.yml

```
@@ -18,7 +18,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: [ '3.0', '2.7', '2.6', '2.5' ]
+        ruby: [ '3.0', '2.7', '2.6' ]
```

We may as well add 3.1 to the list whilst we're at it.

```suggestion
        ruby: [ '3.1', '3.0', '2.7', '2.6' ]
```